src/main/java/com/mineinabyss/mobzy/listener/MobListener.kt

```diff
@@ -167,17 +161,20 @@ object MobListener : Listener {
 
             //if we hit a custom mob, attack or fire an event
             trace?.hitEntity?.toMobzy()?.let { hit ->
-                if (e.leftClicked) {
-                    e.isCancelled = true
+                if (leftClicked) {
+                    isCancelled = true
                     player.toNMS().attack(hit.nmsEntity)
```

Comment:
Have you tested if this doesn't throw any errors if you pass a non-living entity as parameter?

Reply:
NMS code looks like it does checks on its own. I think it wouldn't take an `Entity` class if it didn't work.